observability/logging/logging-6.0/log6x-about.adoc

```diff
@@ -0,0 +1,167 @@
+[id="log6x-about"]
+= Logging 6.0
+:context: logging-6x
+
+toc::[]
+
+The `ClusterLogForwarder` custom resource (CR) is the central configuration point for log collection and forwarding.
+
+== Inputs and Outputs
+
+Inputs specify the sources of logs to be forwarded. Logging provides built-in input types: `application`, `infrastructure`, and `audit`, which select logs from different parts of your cluster. You can also define custom inputs based on namespaces or pod labels to fine-tune log selection.
+
+Outputs define the destinations where logs are sent. Each output type has its own set of configuration options, allowing you to customize the behavior and authentication settings.
+
+
+== Receiver Input Type
+The receiver input type enables the Logging system to accept logs from external sources. It supports two formats for receiving logs: `http` and `syslog`.
+
+The `ReceiverSpec` defines the configuration for a receiver input.
+
+== Pipelines and Filters
+
+Pipelines determine the flow of logs from inputs to outputs. A pipeline consists of one or more input refs, output refs, and optional filter refs. Filters can be used to transform or drop log messages within a pipeline. The order of filters matters, as they are applied sequentially, and earlier filters can prevent log messages from reaching later stages.
+
+== Operator Behavior
+
+The Cluster Logging Operator manages the deployment and configuration of the collector based on the `managementState` field:
+
+- When set to `Managed` (default), the operator actively manages the logging resources to match the configuration defined in the spec.
+- When set to `Unmanaged`, the operator does not take any action, allowing you to manually manage the logging components.
+
+== Validation
+Logging includes extensive validation rules and default values to ensure a smooth and error-free configuration experience. The `ClusterLogForwarder` resource enforces validation checks on required fields, dependencies between fields, and the format of input values. Default values are provided for certain fields, reducing the need for explicit configuration in common scenarios.
+
+=== Quick Start
+
+.Prerequisites
+* Cluster administrator permissions
+
+.Procedure
+
+. Install the `OpenShift Logging` and `Loki` Operators from OperatorHub.
+
+. Create a `LokiStack` custom resource (CR) in the `openshift-logging` namespace:
++
+[source,yaml]
+----
+apiVersion: loki.grafana.com/v1
+kind: LokiStack
+metadata:
+  name: logging-loki
+  namespace: openshift-logging
+spec:
+  managementState: Managed
+  size: 1x.extra-small
+  storage:
+    schemas:
+    - effectiveDate: '2022-06-01'
+      version: v13
+    secret:
+      name: logging-loki-s3
+      type: s3
+    storageClassName: gp3-csi
+  tenants:
+    mode: openshift-logging
+----
+
+. Create a service account for the collector:
++
+[source,shell]
+----
+$ oc create sa collector -n openshift-logging
+----
+
+. Create a `ClusterRole` for the collector:
```

Comment:
we can use  the predefined ClusterRole directly.

Reply:
cluster-logging-write-application-logs
cluster-logging-write-audit-logs  
cluster-logging-write-infrastructure-logs
logging-collector-logs-writer